spec/models/menu_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Menu, type: :model do
+  let(:user) { FactoryBot.create(:user, :user) }
```

Comment:
why two `:user` ?

Reply:
one is referencing the user model and another is the trait, which is defined in FactoryBot